DESCRIPTION

```diff
@@ -37,7 +37,7 @@ Encoding: UTF-8
 LazyData: true
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.3
-Depends: R (>= 3.5)
+Depends: R (>= 4.0)
```

Comment:
I think this is a milestone we should mark in the news that we are no longer saying `admiral` is compatible with `3.5` or `3.6`

Reply:
Totally forgot to add news, I'll add this in as well